src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -157,14 +176,9 @@ private boolean isMatchingByContextStrategy(TreeWalkerAuditEvent event) {
         else if (SUPPORT_CONTEXT_STRATEGY_CHECKS.contains(checkShortName)) {
             final DetailAST eventAst = getEventAst(event);
             final Set<Integer> childAstLineNoList = getChildAstLineNo(eventAst);
```

Comment:
you're still collecting all line numbers instead of min and max.

Reply:
done